configuration/exodus-pipeline.yaml

```diff
@@ -438,10 +477,113 @@ Resources:
       TimeoutInMinutes: 10
     DependsOn: CodeBuildRole
 
+  IntegrationTestsCodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Condition: IsDev
```

Comment:
Currently, this won't ever run, as test stage is only created for stage, prod.

Reply:
It will. This creates the CodeBuild project, which is independent from the CodePipeline, so it doesn't matter which env this resource is created in, so long as it's created in some env before it's referenced (i.e., <= stage). When this template is deployed against dev, the CodeBuild project is created. When the template is deployed against stage and prod, it references the CodeBuild project that was already created in the dev stage. The same method was used for the other CodeBuild project (Condition: !IsDev when in practice, this project is used in both the dev and stage pipelines because we only need one of these to be created). Ideally, we shouldn't use inaccurate "IsDev" conditions to represent or satisfy an "only create this resource once even though this resource is going to be used in multiple envs" condition. A separate template/stack is a better fit for such resources. I've addressed that here: https://projects.engineering.redhat.com/browse/RHELDST-1363 .

Reply:
Ah, right. Yeah...maybe we should have a separate stack of shared resources. This template has become much more complex than it initially was.